aksolve/client/aksolve/minres.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
+"""MINimum RESidual iterative solver for Ax = b"""
+__all__ = ['minres']
+
+
+from typing import Optional, Tuple
+
+import numpy as np
+
+import arkouda as ak
+
+from aksolve.util import eye, EPS, FMAX, inner, norm, Operator
+
+
+def minres(
```

Comment:
Adding the comment here so It is easier overall. I am thinking we may want to break some of these calculations into sub-functions for readability. The functionality looks good to me, but it looks like there are sections that would easily break out.

Reply:
Added Issue #35 to track